data/sql/updates/pending_db_world/rev_1620115611701757811.sql

```diff
@@ -0,0 +1,74 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1620115611701757811');
+
+SET 
+@ROUGHSTONE = 2835,
+@COPPERORE = 2770,
+@COARSESTONE = 2836,
+@TINORE = 2771,
+@HEAVYSTONE = 2838,
+@IRONORE = 2772,
+@SOLIDSTONE = 7912,
+@MITHRILORE = 3858,
+@DENSESTONE = 12365,
+@SOULDARITE = 19774,
+@THORIUMORE = 10620;
+
+-- Copper Vein
+UPDATE `gameobject_loot_template` SET `MaxCount`=6 WHERE FIND_IN_SET (`Entry`,'1502,1735,2626,18092') AND `Item` = @ROUGHSTONE; -- Rough Stone
+UPDATE `gameobject_loot_template` SET `MinCount`=1 WHERE FIND_IN_SET (`Entry`,'1502,1735,2626,18092') AND `Item` = @ROUGHSTONE; -- Rough Stone
+UPDATE `gameobject_loot_template` SET `Chance`=50 WHERE FIND_IN_SET (`Entry`,'1502,1735,2626,18092') AND `Item` = @ROUGHSTONE; -- Rough Stone
+UPDATE `gameobject_loot_template` SET `MinCount`=1 WHERE FIND_IN_SET (`Entry`,'1502,1735,2626,18092') AND `Item` = @COPPERORE; -- Copper Ore
+UPDATE `gameobject_loot_template` SET `MaxCount`=4 WHERE FIND_IN_SET (`Entry`,'1502,1735,2626,18092') AND `Item` = @COPPERORE; -- Copper Ore
+
+-- Tin Vein
+UPDATE `gameobject_loot_template` SET `MaxCount`=6 WHERE FIND_IN_SET (`Entry`,'1503,1736,2627,18093') AND `Item` = @COARSESTONE; -- Coarse Stone
+UPDATE `gameobject_loot_template` SET `MinCount`=1 WHERE FIND_IN_SET (`Entry`,'1503,1736,2627,18093') AND `Item` = @COARSESTONE; -- Coarse Stone
+UPDATE `gameobject_loot_template` SET `Chance`=50 WHERE FIND_IN_SET (`Entry`,'1503,1736,2627,18093') AND `Item` = @COARSESTONE; -- Coarse Stone
+UPDATE `gameobject_loot_template` SET `MinCount`=1 WHERE FIND_IN_SET (`Entry`,'1503,1736,2627,18093') AND `Item` = @TINORE; -- Tin Ore
+UPDATE `gameobject_loot_template` SET `MaxCount`=4 WHERE FIND_IN_SET (`Entry`,'1503,1736,2627,18093') AND `Item` = @TINORE; -- Tin Ore
+
+-- Iron Deposit
+UPDATE `gameobject_loot_template` SET `Chance`=50 WHERE Entry = 1505 AND `Item` = @HEAVYSTONE; -- Heavy Stone
```

Comment:
backtick on entry too, all keywords should have them =)

Reply:
You mean column names, right?

Reply:
yes

Reply:
https://www.azerothcore.org/wiki/sql-standards